### DISCUSSION TITLE
Date-fns on calendar / date picker components

### DISCUSSION THREAD

**Discussion by stefensuhat** [mantinedev/mantine#198] [Feedback]
Hoping date picker will support date-fns in the near future :)

**Comment by rtivital** [comment]
Hi @stefensuhat we use dayjs as date library, what features do you need that are not there?

**Reply by stefensuhat** [reply]
This is more on personal preference :) Wanted to migrate my current ui library to Mantine but most of my config handling for date is based on date-fns.

**Reply by rtivital** [reply]
You can still use Mantine, dayjs is just 2kB, there is nothing wrong in using 2 libraries together

**Reply by stefensuhat** [reply]
yep2. I think I'll do that. thank you @rtivital !

**Reply by omnisci3nce** [reply]
I'm using both as well.
